website/content/en/status/report-2023-01-2023-03/clusteradm.adoc

```diff
@@ -12,22 +12,27 @@ In this quarter, the team has worked on the following:
 * Regular support for FreeBSD.org user accounts
 * Regular disk and parts support (and replacement) for all physical hosts and mirrors
 * Improve the PowerPC package builders +
```

Comment:
I feel that a full stop is needed in places such as this. 

The two sentences that follow do have full stops.